src/services/services.ts

```diff
@@ -3531,7 +3550,8 @@ namespace ts {
         function getSymbolKind(symbol: Symbol, location: Node): string {
             let flags = symbol.getFlags();
 
-            if (flags & SymbolFlags.Class) return ScriptElementKind.classElement;
+            if (flags & SymbolFlags.Class) return ts.forEach(symbol.declarations, d => d.kind === SyntaxKind.ClassExpression) ?

```

Comment:
Maybe just use `firstOrUndefined` and see if that's a class expression; those can't have multiple declarations